### ISSUE TITLE
ros_type_introspection stopped building after the 0.4.0 release into kinetic

### ISSUE THREAD

**Issue by tfoote** [Eurecat/abseil-cpp#5]
The 0.4.0 release triggered the first rebuild that failed. 

https://github.com/ros/rosdistro/pull/21373#issuecomment-500974776

In the later builds I find: http://build.ros.org/view/Kbin_uX32/job/Kbin_uX32__ros_type_introspection__ubuntu_xenial_i386__binary/3/console
```
21:00:48 CMake Error at /opt/ros/kinetic/share/abseil_cpp/cmake/abseil_cppConfig.cmake:150 (message):
21:00:48   Project 'ros_type_introspection' tried to find library 'absl::base'.  The
21:00:48   library is neither a target nor built/installed properly.  Did you compile
21:00:48   project 'abseil_cpp'? Did you find_package() it before the subdirectory
21:00:48   containing its code is included?
21:00:48 Call Stack (most recent call first):
21:00:48   /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:76 (find_package)
21:00:48   CMakeLists.txt:7 (find_package)
```

**Comment by facontidavide** [comment]
I am submitting a fix (0.4.1)

**Comment by tfoote** [comment]
This 0.4.1 doesn't appear to have resolve the issue.

http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__ros_type_introspection__ubuntu_xenial_amd64__binary/10/console
```
16:55:18 -- catkin 0.7.18
16:55:18 -- BUILD_SHARED_LIBS is on
16:55:18 CMake Error at /opt/ros/kinetic/share/abseil_cpp/cmake/abseil_cppConfig.cmake:150 (message):
16:55:18   Project 'ros_type_introspection' tried to find library 'absl::base'.  The
16:55:18   library is neither a target nor built/installed properly.  Did you compile
16:55:18   project 'abseil_cpp'? Did you find_package() it before the subdirectory
16:55:18   containing its code is included?
16:55:18 Call Stack (most recent call first):
16:55:18   /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:76 (find_package)
16:55:18   CMakeLists.txt:7 (find_package)
```
